docs/index.md

```diff
@@ -1,42 +1,13 @@
-Installation Guides
-===================
+Open Science Grid Site Documentation
+====================================
 
-The Open Science Grid consists primarily of a fabric of services at
-participating sites.
+!!! note
+    This is currently a work in progress as we migrate our TWiki documentation to GitHub, so please let us know of any issues that you encounter by [opening a ticket](https://ticket.opensciencegrid.org) or e-mailing [goc@opensciencegrid.org](goc@opensciencegrid.org) with a description of your issue.
```

Comment:
To what does the initial “This” refer? Almost always, the pronouns “this” and “that” should be followed by a noun phrase to indicate the referent; there are exceptions, but you should start with the noun and go from there.

Also, I would give more concrete suggestions than “with a description of your issue” — how about something like (as a new sentence), “Please identify errors, inconsistencies, or incorrect information in the documentation, and also suggest extra content that you feel would be helpful to include in the documentation.”